list1.h

```diff
@@ -0,0 +1,22 @@
+#ifndef LIST1_H
+#define LIST1_H
+#include "list.h"
+class List1
+{
+	private:
+		struct Node
+		{
+			Node* next;
+			int value	
+		};
+	public:
```

Comment:
none of these methods should be implemented here; and they certainly shouldn't have empty implementations.